plan/task/exec.go

```diff
@@ -22,6 +24,7 @@ func init() {
 	Register("Exec", func() Task { return &execTask{} })
 	Register("Start", func() Task { return &asyncExecTask{} })
 	Register("Stop", func() Task { return &stopAsyncExecTask{} })
+	Register("SendSignal", func() Task { return &sendSignalTask{} })
```

Comment:
This file become bigger and bigger after each improvement. If we continue like this, `exec.go` will be unmaintainable.

We should maybe work on a refactor of the file tree to make it more sustainable with that kind of improvement.

However, that should be done in a further PR

Reply:
Agree, I have more PRs pending this one, can update them to split things up here a bit more

Reply:
Great!